internal/acceptance/data_metastores_test.go

```diff
@@ -0,0 +1,16 @@
+package acceptance
+
+import (
+	"testing"
+)
+
+func TestAccDataSourceMetastore(t *testing.T) {
+	workspaceLevel(t, step{
+		Template: `
+		data "databricks_metastores" "this" {}
+		
+		output "all_metastores" {
+			value = data.databricks_metastores.this
+		}`,
+	})
+}
```

Comment:
Using output should be fine, used cluster test for reference: https://github.com/databricks/terraform-provider-databricks/blob/master/internal/acceptance/data_cluster_test.go

Reply:
Tested locally, working fine

Reply:
Also created a run on CI